crates/rspack_binding_api/src/swc.rs

```diff
@@ -57,7 +57,9 @@ fn _transform(source: String, options: String) -> napi::Result<TransformOutput>
   compiler
     .transform(
       source,
-      Some(swc_core::common::FileName::Anon),
```

Comment:
@GiveMe-A-Name transform missing filename cause sourcemap missing sources fields

Reply:
Got it